python/data_extraction/export_data.py

```diff
@@ -23,23 +24,82 @@ def __transform_content(input_filename="data/content.json.gz",
             content_generator = json_arrays.read_json(input_file)
             json_arrays.write_json(output_file, transform_function(content_generator))
 
+def get_all_content():
+    get_content = functools.partial(
+        content_export.get_content,
+        content_store_url=plek.find('draft-content-store')
+    )
+
+    progress_bar = progressbar.ProgressBar()
+
+    content_links_list = list(
+        progress_bar(
+            content_export.content_links_generator(
+                blacklist_document_types=configuration[
+                    'blacklist_document_types'
+                ]
+            )
+        )
+    )
+
+    content_links_set = set(content_links_list)
+    duplicate_links = len(content_links_list) - len(content_links_set)
+
+    if duplicate_links > 0:
+        print("{} duplicate links from Rummager".format(duplicate_links))
+
+    pool = Pool(10)
+    return pool.imap(get_content, content_links_set), len(content_links_set)
```

Comment:
not sure about the second parameter here. The docs say: 

> This method chops the iterable into a number of chunks which it submits to the process pool as separate tasks. The (approximate) size of these chunks can be specified by setting chunksize to a positive integer.

So I think there's only one chunk right? Maybe '1' is better? I just omitted it

Reply:
ah nevermind I can't read